### PR TITLE
Set kubernetes version to 1.13.5

### LINKED ISSUE #826: Set kubernetes version to 1.13 for devel/2.2
Use kubernetes version 1.13 for this branch

### PR DESCRIPTION
**Pull request by ramanduh**
Set kubernetes version to 13.5

Test:
```
[root@bootstrap ~]# kubectl version --kubeconfig=/etc/kubernetes/admin.conf
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.5", GitCommit:"2166946f41b36dea2c4626f90a77706f426cdea2", GitTreeState:"clean", BuildDate:"2019-03-25T15:26:52Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.5", GitCommit:"2166946f41b36dea2c4626f90a77706f426cdea2", GitTreeState:"clean", BuildDate:"2019-03-25T15:19:22Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

Closes #826

### CODE REVIEW

File: container_images.mk [hunk 1]
```diff
@@ -6,21 +6,21 @@ ETCD_IMAGE = k8s.gcr.io/etcd@sha256:b960569ade5f37205a033dcdc3191fe99dc95b15c679
 ETCD_IMAGE_NAME = etcd
 ETCD_IMAGE_VERSION = 3.2.18
```
Comment (alexandre-allard) [inline container_images.mk]: Do we want to upgrade the other images to reflect what's defaulted with this version of Kubernetes ? https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md

Comment (NicolasT) [reply container_images.mk]: Very good point, that's a blocker, and something that wasn't considered in #824 I believe.

**Review by charlesprost — APPROVED** [review]

File: container_images.mk (hunk 1 above)
Comment (TeddyAndrieux) [reply container_images.mk]: etcd get upgraded for kubernetes1.12 so I will do another PR for development/2.1 to upgrade etcd

**Review by TeddyAndrieux — APPROVED** [review]

**Comment by TeddyAndrieux** [comment]
@bert-e approve

**Comment by ramanduh** [comment]
@bert-e approve

**Pull request merged**